src/components/adaptive-cards/Action/Action.jsx

```diff
@@ -37,24 +39,30 @@ export default function Action({
       className={cssClasses}
       isDisabled={isDisabled}
       onClick={onClick}
+      pressed={pressed}
       style={style}
       type={buttonType}
     >
       {data.iconUrl && isValidIconUrl && <img className={sc('icon')} src={data.iconUrl} alt="" />}
       <span>{data.title}</span>
+      {arrowIcon && <Icon className={sc('arrow-icon')} name={pressed ? 'arrow-up' : 'arrow-down'} size={10} />}
```

Comment:
let's use just a `rightIcon` prop that receives the name of the icon (arrow-up or arrow-down or whatever the child component needs) so that the logic is embedded in the child component.